README.md

```diff
@@ -419,6 +419,10 @@ To learn how to get notifications when resource 1 changes, or how to use resourc
 
 As ARM Compiler 5.06 update *3* has multiple compiler bugs (issue 2 above), we recommend updating to [5.06 update 6](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-5).
 
+### UBLOX_EVK_ODIN_W2
+
+Pre-built WiFi binaries are broken for UBLOX_EVK_ODIN_W2 between Mbed OS 5.7.4 to 5.7.x, issue [6003](ARMmbed/mbed-os#6003) (fixed to Mbed OS 5.8.0).
```

Comment:
Would it be better to state "from Mbed OS 5.7.5 to 5.7.x ?

5.7.4 is fine, now the wording is a bit ambiguous?

Reply:
Yep, fixed it like that.